src/Pages/GlobalConfigurations/PluginPolicy/OffendingPipelineModal/OfflinePipelineModalAppView.tsx

```diff
@@ -0,0 +1,91 @@
+import { useHistory, useLocation, useRouteMatch } from 'react-router-dom'
+import { APIResponseHandler, GenericEmptyState, ImageType, noop, useAsync } from '@devtron-labs/devtron-fe-common-lib'
+import noOffendingPipelineImg from '@Images/no-offending-pipeline.svg'
+import { WorkflowCreate } from '@Components/app/details/triggerView/config'
+import { getInitialWorkflows } from '@Components/app/details/triggerView/workflow.service'
+import { Workflow } from '@Components/workflowEditor/Workflow'
+import { OfflinePipelineModalAppViewProps } from './types'
+
+const OfflinePipelineModalAppView = ({ appId, policyKind }: OfflinePipelineModalAppViewProps) => {
+    const history = useHistory()
+    const location = useLocation()
+    const match = useRouteMatch<any>()
+
+    const [isWorkflowsLoading, workflowsResponse, workflowsError, refetchWorkflows] = useAsync(
+        () =>
+            getInitialWorkflows(
```

Comment:
Suggestion: This a heavy call, can we send abortController on unmount, Can we discuss possibility of caching with product?

Reply:
Not required at present